docs/sparql/queries/getAllBundlePopertiesAndTypes

```diff
@@ -0,0 +1,6 @@
+SELECT ?property ?type
+WHERE {
+  <http://www.w3.org/ns/adms#AssetRepository> ?property ?obj.
+  ?property <http://www.w3.org/2000/01/rdf-schema#range> ?type
+}
+GROUP BY (?pred)
```

Comment:
Where is this `?pred` defined? It's clear that it is the predicate, but is this a built-in variable? I couldn't find a reference to it in the SPARQL 1.1 query language specification.